DiceKit/MultiplicationExpressionResult.swift

```diff
@@ -39,3 +39,27 @@ public struct MultiplicationExpressionResult<MultiplierResult: protocol<Expressi
 public func == <M, R>(lhs: MultiplicationExpressionResult<M,R>, rhs: MultiplicationExpressionResult<M,R>) -> Bool {
     return lhs.multiplierResult == rhs.multiplierResult && lhs.multiplicandResults == rhs.multiplicandResults
 }
+
+// MARK: - CustomDebugStringConvertible
+
+extension MultiplicationExpressionResult: CustomDebugStringConvertible {
+    
+    public var debugDescription: String {
+        get {
+            return "\(String(reflecting: multiplierResult)) * \(String(reflecting: multiplicandResults))"
+        }
+    }
+    
+}
+
+// MARK: - CustomStringConvertible
+
+extension MultiplicationExpressionResult: CustomStringConvertible {
+    
+    public var description: String {
+        get {
+            return "\(multiplierResult) * \(multiplicandResults)"
```

Comment:
Ah, I clearly didn't check to make sure this one made sense. I'll fix that in my next push, which should be the one ready for final review.